### ISSUE TITLE
Initial Project Issue Planning

### ISSUE THREAD

**Issue by mpsloan** [mpsloan/CS-7830-Final-Project#1]
- [x] #2 
    - [x] Decide what to do for each features empty cells  
    - [x] Choose best features  
    - [x] Decide what problem we want to solve  
        - [x] Early, mid, late round pick?  
        - [x] Exact round?  
        - [x] Exact round and pick?  
    - [x] EDA  
        - [x] Correlation matrices, etc.  
- [x] #3 
    - [x] One simple model  
        - [x] Logistic regression  
        - [x] KNN  
        - [x] SVM  
        - [x] Random forest  
    - [x] One complex model  
        - [x] MLP  
- [x] #4 
    - [x] Classification report  
        - [x] Precision  
        - [x] Recall  
        - [x] F1  
        - [x] Support  
![Macro average](Attachments/078A2C41-C1DC-4ED4-9CFD-A20178122D7A.png)  
    - [x] Accuracy  
    - [x] Visuals  
        - [x] AUC ROC curve  
        - [x] Confusion matrix  
        - [x] Heat maps  
  
- [x] #5 
    - [x] Pick which simple model we want (what performed best based on our metrics)  
    - [x] Research hyper parameters and potentially modify for better performance  
        - [x] Keep track of these experiments (in the code) to discuss in the report  
- [x] #6 
    - [x] k-Means  
    - [x] Fuzzy c-means  
    - [x] Unsupervised neural networks (deep learning)  
- [x] #7
    - [x] Intro: dataset why?  
    - [x] Dataset preprocessing explanation  
    - [x] Methods: hyperparameters  
    - [x] Results  
    - [x] Analysis/Discussion  
    - [x] Technical Challenges / Next Steps / Limitations
